### PR TITLE
Fix gameserver listing

### PR DESCRIPTION
**Pull request by timiimit**
This PR makes listing hubs and servers within hubs work.

### CODE REVIEW

File: UT4MasterServer/Controllers/UnrealTournamentMatchmakingController.cs [hunk 1]
```diff
@@ -222,6 +242,26 @@ public async Task<IActionResult> ListGameServers([FromBody] GameServerFilter fil
 		return Json(arr);
 	}
 
+
+	/// <summary>
```
Comment (Saibamen) [inline UT4MasterServer/Controllers/UnrealTournamentMatchmakingController.cs:246]:
```suggestion
	/// <summary>
```

File: UT4MasterServer/Controllers/UnrealTournamentMatchmakingController.cs [hunk 2]
```diff
@@ -222,6 +242,26 @@ public async Task<IActionResult> ListGameServers([FromBody] GameServerFilter fil
...
+	/// <param name="showHubs">whether to return hubs</param>
+	/// <param name="showServers">whether to return servers</param>
+	/// <returns></returns>
+	[AllowAnonymous]
+	[HttpGet("session/matchMakingRequest")]
+	public async Task<IActionResult> ListGameServers([FromQuery] bool? showHubs, [FromQuery] bool? showServers)
+	{
+		if (User.Identity is not EpicUserIdentity user)
+		{
+			// allow any third-party project to easily access hub list without any authentication
```
Comment (Saibamen) [inline UT4MasterServer/Controllers/UnrealTournamentMatchmakingController.cs:258]: Maybe add information log?

Comment (timiimit) [reply UT4MasterServer/Controllers/UnrealTournamentMatchmakingController.cs]: Done

File: UT4MasterServer/Program.cs [hunk 3]
```diff
@@ -10,10 +10,13 @@ namespace UT4MasterServer;
 
 public static class Program
 {
+	private static DateTime startupTime = DateTime.Now;
```
Comment (Saibamen) [inline UT4MasterServer/Program.cs:13]:
```suggestion
	private static DateTime startupTime = DateTime.UtcNow;
```

**Review by Saibamen — CHANGES_REQUESTED** [review]

File: UT4MasterServer/Program.cs [hunk 4]
```diff
@@ -10,10 +10,13 @@ namespace UT4MasterServer;
...
+	public static DateTime StartupTime => startupTime;
```
Comment (Saibamen) [inline UT4MasterServer/Program.cs:14]: UTC
```suggestion
	public static DateTime StartupTime { get; } = DateTime.UtcNow;
```

**Review by Saibamen — CHANGES_REQUESTED** [review]

**Pull request merged**